source/MRMesh/MRPointsComponents.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include "MRMeshFwd.h"
+#include "MRUnionFind.h"
+
+namespace MR
+{
+
+namespace PointCloudComponents
+{
+
+/// \defgroup PointClodComponentsGroup PointClodComponents
+/// \ingroup ComponentsGroup
+/// \{
+
+/// get all point cloud components connected by a distance no greater than \param maxDist
+MRMESH_API std::vector<VertBitSet> getAllComponents( const PointCloud& pointCloud, float maxDist );
```

Comment:
This function is too slow and memory inefficient for many components. 

Please write something similar to
```
/// returns the union of connected components, each having at least given area
[[nodiscard]] MRMESH_API FaceBitSet getLargeByAreaComponents( const MeshPart& meshPart, float minArea, const UndirectedEdgePredicate & isCompBd );
```